azkaban-common/src/main/java/azkaban/imagemgmt/daos/RampRuleDao.java

```diff
@@ -0,0 +1,23 @@
+package azkaban.imagemgmt.daos;
```

Comment:
Please add open source and copyright header. Make sure it has correct year. Can be picked from any of the existing code.